desc/geometry/surface.py

```diff
@@ -169,16 +203,18 @@ def change_resolution(self, *args, **kwargs):
         """Change the maximum poloidal and toroidal resolution."""
         assert (
             ((len(args) in [2, 3]) and len(kwargs) == 0)
-            or ((len(args) in [2, 3]) and len(kwargs) in [1, 2])
+            or ((len(args) in [2, 3]) and len(kwargs) in [1, 3])
```

Comment:
len kwargs could be 1,2, or 3 right? for sym, NFP and NFP umbilic? so this should be [1,2,3] instead of [1,3]